onnxruntime/core/providers/cpu/tensor/transpose.cc

```diff
@@ -172,10 +172,41 @@ Status Transpose<float>::Compute(OpKernelContext* ctx) const {
   return Status::OK();
 }
 
-ONNX_CPU_OPERATOR_KERNEL(
+template <>
+Status Transpose<int64_t>::Compute(OpKernelContext* ctx) const {
```

Comment:
Why do we need a specialization for int64? The float one looks exactly the same.

Reply:
It is not exactly the same, it invokes DoTypedTranspose<int64_t> below

---
In reply to: [263208721](https://github.com/Microsoft/onnxruntime/pull/561#discussion_r263208721) [](ancestors = 263208721)